docs/BrandColors.stories.tsx

```diff
@@ -22,6 +26,7 @@ type Story = StoryObj<typeof ColorSwatchGroup>;
 export const Figma: Story = {
   render: () => {
     const { brandColor } = useJsonColor();
+    console.log('brandColor JSON', brandColor);
```

Comment:
Going to retain console logs until we finish the brand evolution work it's quite hepful